### ISSUE TITLE
HIbernating and booting into Windows entirely corrupts linux partition. 

### ISSUE THREAD

**Issue by PoutineSyropErable** [maharmstone/btrfs#708]
Don't use this if you have hibernation.

**Comment by IlgazC** [comment]
The first command required for any dual booting Windows/Linux is running Administrator Command Prompt and issuing` powercfg /h off` command. It isn't related to winbtrfs at all, Windows will happily overwrite Linux partition while hibernating.

**Comment by maharmstone** [comment]
This isn't a bug with the Windows driver. If you hibernate Linux, it's going to assume that any mounted volumes don't change while it's asleep.

>The first command required for any dual booting Windows/Linux is running Administrator Command Prompt and issuing `powercfg /h off` command.

This is good advice, but actually disables the fake shutdowns that Windows does. If you restart rather than shutting down, it always shuts down completely.

>Windows will happily overwrite Linux partition while hibernating.

I am 99% sure that this isn't true.

**Comment by PoutineSyropErable** [comment]
Well, anyway, I removed it to be safer in the future. It was nice to access files from both, saving external usbs and cloud usages for between boot.  
But, as far as I know, no "user controled" write operation happened. We didn't do any write command or use a program and ask it to write. 

I hibernated linux. Left my pc. My dad opened my pc, at the boot manager chose windows. Kept it running while some windows backup happened on an external disk. No read or write operation were manually done by me or him while Arch linux was hibernating from windows.

So, my current best guess is the file indexing process from Everything<Loop Icon> Service or some other decided to do some write operations to metadata. Or some other automatic services. 

On the next boot, this partially corrupted the drive, destroying gnu commands in the terminal and caused a buggy session and the boot after that completely corrupted the @ and @snapshots partition, making the partition unmountable. luckily @home was recoverable to an external drive with some btrfs operation from live environment. Had to reinstall arch and every program and every system config. 

Windows also like to break the boot partition it can detect, making me reinstall and configure grub every weeks or so.This has nothing to do with this driver. But just saying, windows is a grubby asshole who touches what it shouldn't.  

This is more of a warning for users then a possible bug here. It's possible this driver does the same thing but in a different ways, so whatever did a "No operation" write to meta data destroyed it.  

DO NOT HIBERNATE LINUX and then boot into another os!
